<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Savass-MacBook-Air.local" failures="1" tests="3" name="runners.TestRunner" time="51.500" errors="0" timestamp="2025-08-07T23:22:16 GMT+03:00" skipped="0">
  <testcase classname="runners.TestRunner" name="runScenario" time="16.409"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="18.176">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinitions.N11Steps.verifyStorePageOpened(N11Steps.java:32)
at âœ½.User verifies the store page opens correctly(file:///Users/savaskarakus/IdeaProjects/n11-web-automation/src/test/resources/features/n11.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="16.915"/>
</testsuite> <!-- runners.TestRunner -->
